### DISCUSSION TITLE
how to tooltip hide in dynamic way

### DISCUSSION THREAD

**Discussion by akshata456** [spartan-ng/spartan#322] [Q&A]
<img width="486" alt="Screenshot 2024-07-04 at 7 41 04 PM" src="https://github.com/goetzrobin/spartan/assets/34393874/91154931-5a34-464e-9378-484dceb0f3ae">
here i want to close tooltip on click on close icon. tried _brnTooltipTrigger.toggle() , but it not close tooltip immediately

**Comment by ajitzero** [comment]
A popover may be more suitable for your use-case: https://www.spartan.ng/components/popover
